### DISCUSSION TITLE
[Feat]:  Effect cards

### DISCUSSION THREAD

**Discussion by sarussss** [davidjerleke/embla-carousel#719] [Ideas]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [x] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Hi @davidjerleke, I think it would be great if Embla had this feature.

https://swiperjs.com/demos#effect-cards

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment]
@sarussss I can’t see that this request is a duplicate? Can you point me to the other issue where this was requested?

Thanks.

**Reply by sarussss** [reply]
Hi @davidjerleke.
Sorry for the late reply, I don't really understand your question.
https://swiperjs.com/demos#effect-cards
I want to implement this effect for Embla but don't know how.

**Reply by davidjerleke** [reply]
@sarussss I'm not sure why I wrote about a duplicate discussion 😕. Maybe I was very tired and it was past my bedtime 😅. Don't remember. However, This is doable with a bit of tween logic. I will look into it when I get the chance but can't give an ETA unfortunately. The fade plugin and some bugs are prioritised at the time of writing.

Reopening this.

Best,
David

**Reply by sarussss** [reply]
@davidjerleke Thanks a ton.
